src/migrator.js

```diff
@@ -9,8 +9,9 @@ export default class extends Command {
   plugins: Plugins
 
   async run () {
+    if (fs.existsSync(path.join(this.config.dirname, 'plugins', 'package.json'))) return
```

Comment:
@RasPhilCo should this be `path.join(this.plugins.userPluginsDir, 'package.json')`?  Because this looks like it would be a relative path as it is.